pwiz_tools/Skyline/Controls/SeqNode/TransitionTreeNode.cs

```diff
@@ -21,6 +21,7 @@
 using System.Drawing;
 using pwiz.Common.SystemUtil;
 using pwiz.Skyline.Model;
+using pwiz.Skyline.Model.GroupComparison;

```

Comment:
Seems a bit odd that this is suddenly required now. I guess NormalizationMethod must be in here?